src/types.ts

```diff
@@ -27,4 +30,126 @@ export type TileLoadingMonitorOptions = {
   onSlowAlert: () => void
   onLoad: () => void
   onDestroy?: () => void
- }
\ No newline at end of file
+}
+
+//  export type MapOptions = Partial<InstanceType<typeof Map>>
+export type MapOptions = {
```

Comment:
@dadiorchen we created this `MapOptions` for the options you can pass in to the `Map`. Do you know if there are options we have not defined yet?

Reply:
This is a hard question, maybe this is also the reason why we need the type system, I think it's fine if we missed some type here, we can correct it in the future.